api/app/Notifications/PoolCandidateStatusChanged.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Notifications;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Notification;
+
+class PoolCandidateStatusChanged extends Notification
+{
+    use Queueable;
+
+    /**
+     * Create a new notification instance.
+     */
+    public function __construct(
+        public string $old_status,
+        public string $new_status,
+        public string $pool_id,
+        public array $pool_name,
+    ) {
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @return array<int, string>
+     */
+    public function via(object $notifiable): array
+    {
+        return ['database'];
+    }
+
+    /**
+     * Get the array representation of the notification.
+     *
+     * @return array<string, mixed>
+     */
+    public function toArray(object $notifiable): array
+    {
+        return [
+            'old_status' => $this->old_status,
+            'new_status' => $this->new_status,
+            'pool_id' => $this->pool_id,
+            'pool_name' => $this->pool_name,
+        ];
```

Comment:
I'm not sure it matters but do we need to store the pool name here? Seems like it could just add to data bloat since we can use the `pool_id` to get the name where needed.

Reply:
Yeah, I was going back and forth on this.  My reasoning is that including it here should allow the the frontend to render the notification without adding another roundtrip to look up the name.

Reply:
Okay, that makes sense. not like a few extra characters are going to make or break our database storage anyway 😛